.github/workflows/nightly-tool-integrations.yml

```
@@ -99,11 +102,23 @@ jobs:
       AMARU_CHAIN_DIR: ./chain.preprod.db
       AMARU_LEDGER_DIR: ./ledger.preprod.db
       AMARU_LISTEN_ADDRESS: 127.0.0.1:3000
-      AMARU_PEER_ADDRESS: preprod-node.play.dev.cardano.org:3001
+      AMARU_LOG_FILE: ${{ github.workspace }}/amaru.log
+      AMARU_NODE_BINARY: ${{ github.workspace }}/target/release/amaru
+      AMARU_PEER_ADDRESS: ${{ matrix.tool.peer_address || 'preprod-node.play.dev.cardano.org:3001' }}
       AMARU_WITH_OPEN_TELEMETRY: ${{ matrix.tool.metrics || 'false' }}
+      CARDANO_NODE_CONFIG_DIR: ${{ github.workspace }}/cardano-node-config/preprod
+      CARDANO_NODE_DB: ${{ runner.temp }}/db-preprod
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)